### DISCUSSION TITLE
Sublime multi-cursor support

### DISCUSSION THREAD

**Discussion by stalkerg** [zed-industries/zed#16928] [Feature Requests]
Seems like I can't hold `ctrl` and click to set multi-cursor. Do you have plans to support it?

**Comment by Moshyfawn** [comment]
It is indeed supported `alt-click`.

**Reply by stalkerg** [reply]
it doesn't work on Linux/Wayland

**Comment by thataboy** [comment]
In addition, you can change the modifier to ctrl if it's what you're used to. In settings.json


```
  "multi_cursor_modifier": "cmd_or_ctrl",
```

But be aware that a lot of the default key bindings assume you are using alt, the default.

**Reply by stalkerg** [reply]
Thank you! This working for me! I think it should be default behavior for `"base_keymap": "VSCode",`

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
